src/adapter/src/coord/timeline.rs

```diff
@@ -400,7 +400,7 @@ impl Coordinator {
             .expect("no realtime timeline")
             .oracle
             .write_ts(|ts| {
-                self.catalog
+                self._catalog
```

Comment:
There's a couple `_catalog`, are they meant to be `catalog()`?

Reply:
Sadly no, because we're borrowing the timeline in a get_mut, which locks the entire `self` struct. We have to use the member instead of the method because the rust compiler can know that we're not mutating anything else on self.